packages/paste-core/components/card/src/index.tsx

```diff
@@ -1,25 +1,39 @@
 import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {SpacingProps} from '@twilio-paste/types';
+import {StaticDiv} from '@twilio-paste/types/src/DomTypes';
 
-interface CardTitleProps {
-  children: React.ReactNode;
-}
-const CardTitle: React.FunctionComponent<CardTitleProps> = ({children}) => <h4>{children}</h4>;
+interface CardFooterProps extends SpacingProps, StaticDiv {}
 
-interface CardBodyProps {
-  children: React.ReactNode;
-}
-const CardBody: React.FunctionComponent<CardBodyProps> = ({children}) => <div>{children}</div>;
+const CardFooter: React.FunctionComponent<CardFooterProps> = ({children, ...attributes}) => (
+  <Box
+    borderTopWidth="borderWidth10"
+    borderBottomWidth="borderWidth0"
+    borderLeftWidth="borderWidth0"
+    borderRightWidth="borderWidth0"
+    marginTop="space40"
+    paddingTop="space40"
+    borderColor="colorBorder"
+    borderStyle="solid"
+    {...attributes}
+  >
+    {children}
+  </Box>
+);
 
-interface CardProps {
-  onClick: () => void;
-  children: React.ReactNode;
-}
+interface CardProps extends SpacingProps, StaticDiv {}
 
-const Card: React.FunctionComponent<CardProps> = ({children, onClick}) => (
-  // eslint-disable-next-line jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-tabindex
-  <aside onClick={onClick} tabIndex={onClick != null ? 0 : undefined} role={onClick != null ? 'button' : undefined}>
+const Card: React.FunctionComponent<CardProps> = ({children, ...attributes}) => (
+  <Box
+    borderWidth="borderWidth20"
+    borderColor="colorBorder"
+    borderStyle="solid"
+    borderRadius="borderRadius20"
+    padding="space60"
+    {...attributes}
```

Comment:
Not a huge fan of spreading props like this because I can now change everything about the component outside of the systems control. 

Our components should only be modifiable within the confines of the API that we define through variants. If you don't think the component works for you, you can drop down to the `Box` and `Text` layer and create your custom experience.

Reply:
This is definitely a subject to talk about. The current implementation if you're using Typescript will only allow attributes that are defined and extended. This works great if you're using typescript...

If you're not using Typescript you can put anything in there and that sucks. +1

Reply:
If we don't spread then we have to pass through all of the properties (all aria, all events, all etc) which will that hard to maintain Aria, and event passthrough.

Reply:
```typescript 
if (procces.env.TypeChecking) {
    {...attributes}
} else {
  filteredAttributes = filterAttrs(attributes);
  {...filteredAttributes}
}
```

Reply:
Explain the above code snippet to me in more detail, please

Reply:
So if a user opts into TypeChecking by setting up Typescript or fork-ts-checker-webpack-plugin to ensure they have valid typings they can also set an env flag at the startup of the application like...

```js
new webpack.DefinePlugin({
  TypeChecking: JSON.stringify(true),
});
```

once a user has "opted in" to Typechecking then we can take the "easy path" we can use Types as a source of truth and drop a lot of the runtime checking and just passthrough attributes that match closer to the types that are coming from the React types package.

Another thought we should probably drop runtime checking when `process.env.NODE_ENV === 'production'` to mirror React's functionality and dropping development warnings when run in production mode

Reply:
OK, so given no one really has a good answer to solve this right now, we're making the decision not to solve it just yet, in favour of being explicit. 

We as component authors should have a reasonable idea of what is appropriate to add to an element from semantic and accessibility point of view given the intended use case of the component/design. This should inform us to declare a set of props a consumer would likely need. 

This will be plenty good enough to unblock us and get rolling.